src/main/java/abex/os/keepassxc/KeePassXcPanel.java

```diff
@@ -107,6 +110,16 @@ public void open(GetLogins.Response logins)
 		SwingUtil.fastRemoveAll(this);
 		setLayout(new DynamicGridLayout(0, 1, 0, 0));
 		boolean hideUsernames = client.getPreferences().getHideUsername();
+
+		if (config.defaultFirstEntry())
+		{
+			GetLogins.Entry e = logins.getEntries().get(0);
+			client.setPassword(e.getPassword());
+			client.setUsername(e.getLogin());
+			close();
+			return;
```

Comment:
Removing the `return` and `close()` statements will create the behavior you describe. The behavior with the statements is that the panel never even gets to open.

Reply:
I see, I missed the `close()` call. Unfortunately it would likely take a lot of refactoring to move this call before the panel open call, which makes this a reasonable spot to do it. 

However, I still think it would be nicer to have the panel still open, in case the user either
a) clears the information and wants to bring it back
b) wants a default login for a main account, but has multiple

Of course, Abex can take it from here, I just follow the repo for no reason :P

Reply:
I see, I missed the `close()` call. Unfortunately it would likely take a lot of refactoring to move this call before the panel open call, which makes this a reasonable spot to do it. 

However, I still think it would be nicer to have the panel still open, in case the user either
a) clears the information and wants to bring it back
b) wants a default login for a main account, but has multiple

Of course, Abex can take it from here, I just follow the repo for no reason :P

Reply:
I updated it to keep the panel open and just populate the values. I appreciate the feedback 😄 
[Example](https://i.imgur.com/MNhva8J.gif)